requirements/common.txt

```diff
@@ -9,6 +9,7 @@ djangorestframework-expiring-authtoken==0.1.4
 django-cors-headers==1.3.1
 django-rest-auth[with_social]
 drfdocs==0.0.11
+pathlib==1.0.1
```

Comment:
Why you have added this? @viditjain08

Reply:
Travis Build was failing because there was no module named pathlib, and it's failing now as well